src/main/java/page_objects/TicketPricePage.java

```diff
@@ -0,0 +1,19 @@
+package page_objects;
+
+import helper.BrowserHelper;
+import org.openqa.selenium.By;
+
+public class TicketPricePage extends GeneralPage {
+
+    //Elements
+    private final String dynamicBtnCheckPriceXpath = "//li[text() = '%s to %s']//ancestor::tr//a";
```

Comment:
not a good xpath

Reply:
thank you for the review, I will find better XPath